src/components/visits/VisitsDetailsComponent.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <section class="visitDetails">
+        <h3 class="visitDetails__header">Podsumowanie wydarzenia:</h3>
+        <article class="visitDetails__infoBox infoBox">
+            <h5 class="infoBox__label">Tytuł:</h5>
+            <p class="infoBox__description">{{ title }}</p>
+        </article>
+
+        <article class="visitDetails__infoBox infoBox">
+            <h5 class="infoBox__label">Data:</h5>
+            <p class="infoBox__description">{{ date }} {{ hour }}</p>
+        </article>
+
+        <article class="visitDetails__infoBox infoBox">
+            <h5 class="infoBox__label">Miejsce:</h5>
+            <p class="infoBox__description">{{ place }}</p>
+        </article>
+
+        <article class="visitDetails__infoBox infoBox">
+            <h5 class="infoBox__label">Lekarz:</h5>
+            <p class="infoBox__description">{{ doctor }}</p>
+        </article>
+    </section>
+</template>
+
+<script setup>
+    defineProps({
+        title: {
+            type: String,
+            default: '',
+        },
+        date: {
+            type: String,
+            default: '',
+        },
+        hour: {
+            type: String,
+            default: '',
+        },
+        place: {
+            type: String,
+            default: '',
+        },
+        doctor: {
+            type: String,
+            default: '',
+        },
+    });
+</script>
+
+<style lang="scss" scoped>
+    .visitDetails {
+        @include flex-position(column, nowrap, flex-start, flex-start);
+        height: 100%;
+        gap: 10%;
+
+        &__header {
+            @include text-header3($font-weight-semiBold);
+            align-self: center;
+            color: $blue-900;
+        }
+
+        &__infoBox {
+            @include flex-position(row, wrap, space-between, center);
```

Comment:
jesli tu jest space between to tamte zmiany sa niewazne, bo jak rozumiem to samo pozycjonuje co nie?

Reply:
which changes? How it 'self' postioning?

Reply:
okej, to bylo do zmian wyzej, jak cos jest z "preetier" to luz, approved.